### ISSUE TITLE
BUG: editing streamdeck dcs-bios button action causes error

### ISSUE THREAD

**Issue by matt-cahill** [DCS-Skunkworks/DCSFlightpanels#65]
```
02.05.2020 10:44:27  version : 1.0.218.6063

System.NullReferenceException
Object reference not set to an instance of an object.
   at DCSFlightpanels.Windows.DCSBiosInputWindow.ComboBoxInterfaceType_OnSelectionChanged(Object sender, SelectionChangedEventArgs e)
```
![](https://cdn.discordapp.com/attachments/700674160304848906/705913178428669992/unknown.png)

**Comment by matt-cahill** [comment]
this can be reproduced by doing the following

1. start a fresh profile
2. add any dcs-bios action to a button
3. exit the dialog
4. accept changes for the button
5. click add/edit
6. select the action and click edit
7. search for a new control
8. click any control from the list of found controls

**Comment by matt-cahill** [comment]
confirmed fixed in 4.0.30
